src/app/api/organizations/[organizationId]/route.ts

```diff
@@ -24,3 +27,32 @@ export async function DELETE(
   }
   return new NextResponse(undefined, { status: 204 });
 }
+
+// @route PUT /api/organizations/[organizationId] - updates an existing organization in the database
+export async function PUT(
```

Comment:
Was the call tested through postman/MongoDB?

Reply:
yes, for both

Reply:
It worked well when I tested them, but did it not work?

Reply:
yes, for both; It worked well when I tested them, but did it not work?

Reply:
Sounds good then. You had left the "I have added tests that prove my fix is effective or that my feature works" blank, so I was curious.